includes/class-wc-payments.php

```diff
@@ -306,6 +310,8 @@ public static function init() {
 			new WC_Payments_Status( self::get_wc_payments_http(), self::get_account_service() );
 		}
 
+		new Platform_Checkout_Tracker( self::get_wc_payments_http() );
```

Comment:
Should we include and instantiate this class within the `WC_Payments_Features::is_platform_checkout_enabled()` conditional above in line 224, like we're doing for `Platform_Checkout_Save_User`? Thinking that this would only be used in those cases.

Reply:
Good point @a-danae, adjusted with https://github.com/Automattic/woocommerce-payments/pull/3796/commits/26d5b45b67a2ddbc4e2a8e4701abbbd39da8d74e

Reply:
Thanks!